distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -17,6 +17,11 @@
 #
 ################################################################################
 
+# In order to switch to Log4j2 and be able to take advantage of the newer appenders,
```

Comment:
Looks like Karaf 4.1.0 is moving to Log4j2.  Might be worth creating a JIRA ticket to upgrade to that version and comment here to remove after upgrade Karaf to that version.

https://issues.apache.org/jira/browse/KARAF-4189
https://github.com/apache/karaf/commit/39d2035b2942dc5791a6e4c63fb969fa23e88636#diff-6ad2eb21294e8aed49b869895b30138b